src/System Application/App/Security Groups/src/InheritedPermissionSetsPart.Page.al

```diff
@@ -0,0 +1,116 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Security.AccessControl;
+
+page 9821 "Inherited Permission Sets Part"
```

Comment:
do the app.json ranges need to be updated?

Reply:
Done thanks